src/methods_segmentation/watershed/config.vsh.yaml

```diff
@@ -0,0 +1,179 @@
+name: watershed
+label: "watershed Segmentation"
+summary: "Output of the segmantation methot watershed"
+description: "Output of the segmantation methot watershed"
+links:
+  documentation: "https://github.com/openproblems-bio/task_ist_preprocessing"
+  repository: "https://github.com/openproblems-bio/task_ist_preprocessing"
+references:
+  doi: "10.1109/ICAA53760.2021.00010"
```

Comment:
The original publication of the algorithm should be provided:

[10.1109/34.87344](https://doi.org/10.1109/34.87344)

https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=87344